libmscore/sym.cpp

```diff
@@ -5035,8 +5035,8 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       "Mute on",
       "Overpressure, down bow",
       "Overpressure, no bow direction",
-      "Overpressure possibile, down bow",
-      "Overpressure possibile, up bow",
+      "Overpressure possible, down bow",
+      "Overpressure possible, up bow",
```

Comment:
Let these stand. Italian.

Reply:
Also a string we inherited from SMuFL, so can't change lightly